openscap_report/html_report/templates/css/patternfly.css

```diff
@@ -1,3 +1,8 @@
+/*!
```

Comment:
Are you going to update this file manually every time you update PF? I'd say that if you want to add license information for an external file, it's better to have it as a separate file. Like `patternfly-license.txt` or something like that.